### ISSUE TITLE
JAWS announces decorative shapes in Microsoft PowerPoint

### ISSUE THREAD

**Issue by dabrams888** [FreedomScientific/standards-support#569]
### Summary

> MS PowerPoint has a feature in the alt text pane which allows you to mark certain objects as decorative for users reading the presentation with a screen reader. The intent is for screen readers to ignore objects with this feature enabled. However, JAWS still announces "decorative shape" or similar when interacting with objects marked as decorative. Other screen reader software such as NVDA properly ignores these objects.
> 
> <img width="110" alt="Mark as decorative checkbox in PowerPoint's alt text pane" src="https://user-images.githubusercontent.com/3267932/129919193-f2b2e97e-abc7-4d3b-bf20-8c00f2cc96ec.PNG">
> 
> Example:
> 1. Open the attached .pptx file
> 2. View the file in presentation mode
> 3. With JAWS enabled, read through the slide content

### Expected result

> JAWS should announce the slide heading, "Step 1", and "Step 2"

### Actual result

> JAWS announces the slide heading, a decorative shape, "Step 1", and "Step 2"

### Example

> [Decorative.pptx](https://github.com/FreedomScientific/VFO-standards-support/files/7008116/Decorative.pptx)

#### JAWS version and build number
JAWS Version 2021.2102.34 ILM

 
#### Operating System and version
Windows 10 Pro Version 10.0.19042 Build 19042

#### Software and version:
Microsoft PowerPoint for Microsoft 365 MSO (16.0.14228.20216) 64-bit Version 2107 Build 14228.20250

**Comment by menovak** [comment]
hi dabrams888,

I will ask a colleague to test in Windows, but when I load the example pptx in powerpoint on the Mac, and test with Voice Over, it doesn't seem to matter if I check the 'mark as decorative' checkbox or not, the line segment which connects Step 1 and Step 2 always announces as _decorative_ with Voice Over.

Stay tuned...

Mark

**Comment by dabrams888** [comment]
Quick update since this hasn't had activity for a while. We had a screen reader user comment today that JAWS was not ignoring decorative items in PowerPoint. So it looks like this issue is still relevant in the current version of JAWS.

**Comment by stevefaulkner** [comment]
hi @dabrams888 have discussed with @BrettLewisVispero and its the internal backlog,  please ping Brett on teams internally for an update
